pom.xml

```diff
@@ -0,0 +1,98 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" 
+		xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
```

Comment:
I think we are missing the deployer resource docker dependency.   
		<dependency>
			<artifactId>spring-cloud-deployer-resource-docker</artifactId>
			<groupId>org.springframework.cloud</groupId>
		</dependency>

Reply:
it should be brought in transitively from the Kubernetes deployer. I added a `DelegatingResourceLoader` @Bean to resolve this ServletContext problem

Reply:
Using latest version resolves issue.